srcpkgs/reddio/reddio.INSTALL.msg

```diff
@@ -0,0 +1 @@
+Optional: nc to authenticate with your Reddit account credentials
```

Comment:
Please add some names of packages that contain nc.

Reply:
Or just drop the INSTALL.msg. It seems that most people don't read them anyway.
I think we should use them for really important stuff only, so there is a higher signal/noise ratio and a better chance of people reading them.